lib/mayaHydra/hydraExtensions/adapters/materialNetworkConverter.cpp

```diff
@@ -657,6 +762,21 @@ void MayaHydraMaterialNetworkConverter::initialize()
                 { MayaHydraAdapterTokens->opacity, transmissionToOpacity },
                 { MayaHydraAdapterTokens->metallic, metallicConverter },
             } } },
+        { MayaHydraAdapterTokens->openPBRSurface,
+          { UsdImagingTokens->UsdPreviewSurface, // Maya OpenPBR surface shader translated to a
+                                                 // UsdPreviewSurface with the following
+                                                 // UsdPreviewSurface parameters mapped
+            {
+                { MayaHydraAdapterTokens->diffuseColor, openPBRBaseColorConverter },
+                { MayaHydraAdapterTokens->emissiveColor, openPBREmissionColorConverter },
+                { MayaHydraAdapterTokens->specularColor, openPBRSpecularColorConverter },
+                { MayaHydraAdapterTokens->ior, openPBRSpecularIORConverter },
+                { MayaHydraAdapterTokens->roughness, openPBRSpecularRoughnessConverter },
+                { MayaHydraAdapterTokens->clearcoat, openPBRCoatConverter },
+                { MayaHydraAdapterTokens->clearcoatRoughness, openPBRCoatRoughnessConverter },
+                { MayaHydraAdapterTokens->opacity, openPBRTransmissionToOpacity },
+                { MayaHydraAdapterTokens->metallic, openPBRMetallicConverter },
```

Comment:
I think you should set the useSpecularWorkflow as well :
{ MayaHydraAdapterTokens->useSpecularWorkflow, fixedOneInt }, so the specular color is used :

useSpecularWorkflow - int - 0 (by default)

This node can fundamentally operate in two modes : Specular workflow where you provide a texture/value to the specularColor input. Or, Metallic workflow where you provide a texture/value to the metallic input. Depending on the 0 or 1 value of this parameter, the following parameters are conditionally enabled:

Reply:
hmm, why we didn't do that for standardSurface here? I remember we handled that logic somewhere else?

Reply:
I saw it, it was already handled here for all material converters

        // If we are using a specular color which is not white, the UsdPreviewsurface specular
        // workflow must be enabled to use the specular color which is done by setting the
        // UsdPreviewSurface param "useSpecularWork" to 1
        {
            const auto it = material.parameters.find(_specularColorToken);
            if (it != material.parameters.cend()) {
                const VtValue& specColorVal = it->second;
                if (!specColorVal.IsEmpty()
                    && specColorVal.UncheckedGet<GfVec3f>() != GfVec3f(1, 1, 1)) {
                    material.parameters[_useSpecularWorkflowToken] = VtValue(1);
                }
            }
        }